package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@sentry/react": "^9.1.0",
```

Comment:
do you remember the reason for adding this?
pretty sure we don't need package.json in root directory at all...